### DISCUSSION TITLE
Background in the tickets

### DISCUSSION THREAD

**Discussion by Hekatonchejr** [alfio-event/alf.io#1309] [Q&A]
Hi, I wanted to change background image in body's style in the template of ticket's PDF. I was trying to do it by changing CSS background-image atribute and even by adding image by HTML's tags - nothing work. I Assume that these feature is blocked for some reason, but I receive ticket layout to upload and I need full-operational stylesheet do to it. How can I change background-image in ticket's template, do you have any advice? I Would be very gratefull

**Comment by cbellone** [comment]
Hi!
we don't allow loading external images for security and performance reasons.
What you can do is embed the image as base64 using the `data:` format, see https://www.w3docs.com/snippets/html/how-to-display-base64-images-in-html.html
